### ISSUE TITLE
two different links leading to step 8

### ISSUE THREAD

**Issue by RheuX** [open-learning-exchange/open-learning-exchange.github.io#3396]
### Please verify the following:

- [X] This issue is related to the [Virtual Interns First Steps](https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md) document or [Planet User Manual](https://open-learning-exchange.github.io/#!pages/manual/planet/overview.md).
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Avi) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md

### Problem

I found that the link from step 7 to step 8 is this: https://open-learning-exchange.github.io/#!./pages/vi/vi-create-issues-and-pull-requests.md

And if you're navigation from the first steps, the link is as follows: https://open-learning-exchange.github.io/#!./pages/vi/vi-issues-and-prs.md

The second link has more detailed on what issues we need to create, while the first one is more generic? I'm not sure which one is more updated, and also for people only follows the next link from each step, they may miss this.

### Screenshots (when necessary)

First Link:
<img width="1216" alt="Screenshot 2024-10-10 at 3 19 48 PM" src="https://github.com/user-attachments/assets/21ce465f-7650-4dd5-8b31-3d4725fbc24f">

Second Link:
<img width="1192" alt="Screenshot 2024-10-10 at 3 19 37 PM" src="https://github.com/user-attachments/assets/1b4f1e9f-ea5f-412e-9181-45225a2a7f00">

### Proposed solution(s)

Changed both link to the same file, but I'm not sure which one is the intended file; Since one is more specific (requiring `issues` from specific manual/pages, and one is more generic, but they all asked for the same thing at the end.

**Comment by xyb994** [comment]
Aha, good catch!

`vi-create-issues-and-pull-requests.md` is older and I did not realize we had that file. But I like the sectional format on this one better and was having trouble coming up with one when I made `pages/vi/vi-issues-and-prs.md`

I think it make sense to keep `Introduction` and `Introduction` section in `vi-create-issues-and-pull-requests.md`, but move majority of the content in `vi-issues-and-prs.md`to `TODO` (perhaps there's a better name for this?) section in `vi-create-issues-and-pull-requests.md`.

Go ahead and make a pull request for this!
